EffectivePotential/include/effectivepotential/one_loop_potential.hpp

```diff
@@ -74,6 +74,10 @@ class OneLoopPotential : public Potential {
   PROPERTY(double, renormalization_scale, 246.)
   /** The gauge parameter \xi */
   PROPERTY(double, xi, 0.)
+  /** Treatment of the thermal masses **/
+  /** false: Arnold-Espinosa  method **/
+  /** true: Parwani method **/
+  PROPERTY(bool, use_Parwani_method, false)
```

Comment:
an enum might be better, so we can do `set_method(ArnoldEspinosa)`, `set_method(Parwani)`, `set_method(SomethingInTheFuture)`?

Reply:
Should we call it 
set_daisy_method
or 
set_daisy_correction_method
or 
set_thermal_mass_method?